index.html

```diff
@@ -10,10 +10,20 @@
     <div class="page">
         <!--header-->
         <div class="header">
+            <div class="left">
             <div class="branch">
                 <img src="logo.png" alt="" class="logo" />
                 <span class="name"> Prechu </span>
             </div>
+            </div>
+            <div class="navigation">
+                <ul>
```

Comment:
what is the role of ul in navigation context? if it does not play any role, it means, we can remove it